src/main/docs/guide/kotest5/kotest5ConstructorInjectionCaveats.adoc

```diff
@@ -0,0 +1,5 @@
+There are a couple caveats to using constructor injection to be aware of.
+
+1. In order for `TestPropertyProvider` to work, test classes must have not have any constructor arguments. This is because the class needs constructed prior to bean creation in order to add the properties to the context. Fields and methods will still be injected.
```

Comment:
```suggestion
1. In order for `TestPropertyProvider` to work, test classes must have not have any constructor arguments. This is because the class needs to be constructed prior to bean creation, in order to add the properties to the context. Fields and methods will still be injected.
```